source/includes/_categories.md

```diff
@@ -49,7 +49,7 @@ Use this endpoint to get a flattened list of all categories in alphabetical orde
       "exclude_from_totals": true,
       "updated_at": "2020-01-28T09:49:03.238Z",
       "created_at": "2020-01-28T09:49:03.238Z",
-      "is_group": false,
+      "is_group": true,
```

Comment:
thanks! perhaps line 39 `"is_group": true` should be `false` then?

Reply:
I guess so! Didn't notice it before.